ui/assets/css/main.css

```diff
@@ -249,3 +249,11 @@ pre {
   white-space: pre-wrap;
   word-break: keep-all;
 }
+
+.search-bar {
+  flex-basis: 10%;
+  flex-shrink: 3;
+  flex-grow: 0.3;
+  max-width: 33%;
+  min-width: 11em;
+}
```

Comment:
This flex-search is nice, but before I'd merge a search change, I'd really rather it be an expanding button ala the ones shown in this example here: https://github.com/LemmyNet/lemmy/issues/814 .

This one just takes up too much space on the screen.

Another thing: I think the reason nearly all examples show a search button next to the input, is for accessibility reasons, as well as touch search. It'd be good to have one anyway, especially after adding the expanding search.

Reply:
Example of bootstrap's suggested expanded one: 
![image](https://user-images.githubusercontent.com/930215/86195291-869bec80-bb1e-11ea-98b1-d9d902bbda3d.png)

Reply:
done